_rules/document-title-is-descriptive.md

```diff
@@ -0,0 +1,165 @@
+---
+name: Document title is descriptive
+rule_type: atomic
+description: |
+  This rule checks that the document title describes the topic or purpose of the document.
+
+success_criterion: 
+- 2.4.2 # Page Titled (A)
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Anne Thyme Nørregaard
+- Corbb O’Connor 
+
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to the first [non-empty](#non-empty) HTML `title` element in a [document](https://www.w3.org/TR/dom/#concept-document) where the [document element](https://www.w3.org/TR/dom/#document-element) is an HTML `<html>` element.
+
+### Expectation
+
+Each target element describes the topic or purpose of the overall content of the [document](https://www.w3.org/TR/dom/#concept-document).
+
+## Assumptions
+
+- This rule assumes that browsers only recognize the first `title` element if multiple `title` elements are present in the [document](https://www.w3.org/TR/dom/#concept-document). Testing shows that this in general is the case. Therefore the scope of this rule is limited to only checking the first `title` element in a document.
```

Comment:
Can you source this testing or is this done by the auto-wcag community?

Is the other applicability rule (that most browsers skip empty title elements) also tested? If so, can this be added to the assumption?

Reply:
Done by Auto-WCAG for this other rule: https://auto-wcag.github.io/auto-wcag/rules/SC2-4-2-page-has-title.html 

And I can see now that I got the applicability slightly wrong. To match the other rule, it has to be: 

> This rule applies to the first HTML `title` element in a [document](https://www.w3.org/TR/dom/#concept-document) where the [document element](https://www.w3.org/TR/dom/#document-element) is an HTML `<html>` element, if the first HTML `title` element is [non-empty](#non-empty).